Replace 'check_screenlock' by 'ensure_unlocked_desktop'

okurz opened the pull request:
Ensure the screen stays unlocked and not only check. Otherwise the screen
might lock immediately after checking.

Verification run: http://lord.arch/tests/3098

Related progress issue: https://progress.opensuse.org/issues/13258

okurz commented:
should also fix https://openqa.suse.de/tests/524122#step/xterm/7

nilxam commented:
isn't it can be simple just add a **send_key 'esc';** after https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L335 ? it's fine though, but the commit subject should change to a correspond one - it's renamed check_screenlock actually...

okurz commented:
@nilxam yes, I renamed. Basically the change I did *is* just a `send_key 'esc'`. Additionally I changed "check_screen" to "assert_screen" because otherwise the whole block would be skipped.

Soulofdestiny commented:
LGTM
